src/components/auth/Auth.js

```diff
@@ -35,6 +37,18 @@ class Auth extends React.Component<Props> {
   }
 
   async componentWillMount() {
+    const { navigation } = this.props
+    const mnemonic = await AsyncStorage.getItem(GD_USER_MNEMONIC)
```

Comment:
if user has mnemonic he will never reach this see index.js. see above requested change to move this logic to the SignupRouter

Reply:
done